conjur/api/client.py

```diff
@@ -14,15 +14,15 @@
 import netrc
 
 from Utils.utils import Utils
-from conjur.api import Api
+from conjur.api.api import Api
 from conjur.config import Config as ApiConfig
 from conjur.constants import DEFAULT_NETRC_FILE
-from conjur.init.init_controller import InitController
-from conjur.init.init_logic import InitLogic
-from conjur.init.conjurrc_data import ConjurrcData
-from conjur.credentials_from_file import CredentialsFromFile
+from conjur.controllers.init_controller import InitController
+from conjur.logics.init_logic import InitLogic
```

Comment:
`logics` isn't a word.. seems a bit weird. So I would maybe write logic

Reply:
maybe leave everything singular. for example, api isn't apis

Reply:
Will change to singular

Reply:
changed to singular and logic